CHANGELOG.md

```diff
@@ -7,6 +7,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [unreleased]
```

Comment:
- size check failed
- Is this change fully backwards compatible? If so, please update patch version, else we can discuss